src/pages/index.js

```diff
@@ -98,4 +99,30 @@ const Index = () => {
   );
 };
 
+export const getStaticProps = async () => {
+  const fallbackUserCount = 200000;
```

Comment:
Minor nitpick: Would probably be easier to read with separators (assuming it's supported)
```suggestion
  const fallbackUserCount = 200_000;
```

Reply:
Good shout; updated that. 

I also changed the build command to `yarn start` instead of `yarn build` in the netlify.toml file, that way we can test the ISR on this PR